geomesa-convert/geomesa-convert-scripting/src/test/scala/org/locationtech/geomesa/convert/scripting/ScriptingFunctionFactoryTest.scala

```diff
@@ -0,0 +1,96 @@
+/***********************************************************************
+  * Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+  * All rights reserved. This program and the accompanying materials
+  * are made available under the terms of the Apache License, Version 2.0
+  * which accompanies this distribution and is available at
+  * http://www.opensource.org/licenses/apache2.0.php.
+  *************************************************************************/
```

Comment:
going to have to remove the leading spaces for the license check to work...
hmm, I might be able to fix it so that it will accept either.